@ui/design/settings/commonAuth.module.scss

```diff
@@ -4,7 +4,7 @@
 .authContainer {
   background-color: colors.$dark-700;
   width: 100%;
-  height: calc(100vh - 60px);
+  height: calc(100vh + 60px);
```

Comment:
Why +60px?
this container =  height of the body, not including the header
& we don't need scrolling if we don't need it